packages/listener-solidarity/src/utils/geocoding.ts

```diff
@@ -149,5 +149,5 @@ export default async ({
   const s = state ? `${state},` : "";
   const z = cep ? cep + ",BR" : "";
 
-  return await getGoogleGeolocation(a + c + s + z, email);
+  return await getGoogleGeolocation(a + n + c + s + z, email);
```

Comment:
@vivianedias sugiro usar nomes mais explícitos para variáveis, se eu ler o código restante eu vou conseguir entender, porém se eu leio apenas essa linha eu já não consigo entender o que acontece, evite sempre nomear variáveis com termos pouco convencionais, se for um caso de organização tente usar Objeto ou dicionário que represente o que está sendo feito

Reply:
add uma variavel chamada "composeSearchAddress" que clarifica a soma dessas letras! acho q fica assim fica mais claro :)